telemetry/telemetrybuffer_test.go

```diff
@@ -71,7 +71,7 @@ func TestWrite(t *testing.T) {
 	_, closeTBServer := createTBServer(t)
 	defer closeTBServer()
 
-	tbClient := NewTelemetryBuffer()
+	tbClient := NewTelemetryBuffer(nil)
```

Comment:
one test case with logger ?

Reply:
yes I agree we should have one test case with this logger, let me change this